packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -76,10 +77,15 @@ def test_if_uiconfig_is_available(cluster):
 
 
 def test_if_dcos_history_service_is_up(cluster):
-    r = cluster.get('/dcos-history-service/ping')
 
-    assert r.status_code == 200
-    assert 'pong' == r.text
+    @retry(stop_max_delay=20000, wait_fixed=500)
```

Comment:
@cmaloney ok, when considering scope, I agree the wait_for_up issue and 'make sure endpoints on all agents and masters' are much bigger than this PR, so I'm ok with dropping them.

However, there is still the fact this test pulls an endpoint that doesnt pull state. So, just hitting /dcos/history-service/last and verifying non-empty would cover that.